docs/guides/centos-installation.sh

```diff
@@ -0,0 +1,96 @@
+################################
+############ HackMD ############
+################################
+
+# Create your HackMD passphrase
+echo "Create your HackMD passphrase for the MySQL database and press [Enter]. You will create your specific HackMD credentials after the installation."
+read -s hackmdpassphrase
+
+# Set your IP address as a variable. This is for instructions below.
+IP="$(hostname -I | sed -e 's/[[:space:]]*$//')"
+
+# Install dependencies
+sudo yum install epel-release -y
+sudo yum install mariadb-server npm gcc-c++ git bzip2 -y
+
+# Stage HackMD for building
+sudo npm install -g uws node-gyp tap webpack grunt yarn
+sudo yarn add -D webpack-cli
+sudo git clone https://github.com/hackmdio/hackmd.git /opt/hackmd/
+cd /opt/hackmd
```

Comment:
When you create the directory and chown it to the current user before you clone it, you can minimize the attack surface.

```
sudo mkdir /opt/hackmd
sudo chown $(id -u):$(id -g) /opt/hackmd
git clone https://github.com/hackmdio/hackmd.git /opt/hackmd
cd  /opt/hackmd
bin/setup
```